companies/3dhubs.json

```diff
@@ -9,7 +9,7 @@
     "name": "3D Hubs B.V.",
     "address": "Frederiksplein 42\n1017 XN Amsterdam\nNetherlands",
     "phone": "+31 20 261 1900",
```

Comment:
They seem to have also changed their address and phone number. Would you mind updating those as well, while you're at it? :)

Reply:
Sure.